packages/fx-core/src/component/driver/botAadApp/create.ts

```diff
@@ -44,6 +44,7 @@ export class CreateBotAadAppDriver implements StepDriver {
       const createRes = await botRegistration.createBotRegistration(
         context.m365TokenProvider,
         args.name,
+        args.name,
```

Comment:
Passing anything here doesn't matter too much because in remote bot registration name is controlled by bicep. The name passed here will not be used if you dive deep into the code.

Reply:
I think it's better to refine the parameter list rather than passing a useless parameter.

Reply:
Agree - note it in future improvements.

Reply:
I have the same feeling, let's talk about it to define a better interface for this case.